From 63ce58c3fa57a4322ab72082005aa12c08943b4e Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Tue, 17 Apr 2018 11:28:03 +0800
Subject: [PATCH] arm64: dts: rockchip: clean up output ctl gpios for rk3308

Add support hp-ctl gpio for amic board, and rename
to spk-ctl from dtsi file. Usually, we should assign
these output ctl gpios according to the specific
product type.

Change-Id: Icf0c29f61bed3c0f48069b9e38ddf47d921fed26
Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts | 3 +++
 arch/arm64/boot/dts/rockchip/rk3308.dtsi             | 1 -
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts b/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts
index 804e8d2790aa..146e130bd87f 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts
@@ -30,6 +30,9 @@
 	status = "okay";
 
 	#sound-dai-cells = <0>;
+
+	hp-ctl-gpios = <&gpio0 RK_PA1 GPIO_ACTIVE_HIGH>;
+	spk-ctl-gpios = <&gpio0 RK_PA5 GPIO_ACTIVE_HIGH>;
 };
 
 &i2s_8ch_2 {
diff --git a/arch/arm64/boot/dts/rockchip/rk3308.dtsi b/arch/arm64/boot/dts/rockchip/rk3308.dtsi
index bbdc461eeae4..cdf5505682a5 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3308.dtsi
@@ -934,7 +934,6 @@
 		clock-names = "acodec", "mclk_tx", "mclk_rx";
 		resets = <&cru SRST_ACODEC_P>;
 		reset-names = "acodec-reset";
-		spk_ctl-gpios = <&gpio0 RK_PA5 GPIO_ACTIVE_HIGH>;
 		status = "disabled";
 	};
 
-- 
2.35.3

